021-server-side-rendering/021-server-side-rendering.md

```diff
@@ -0,0 +1,296 @@
+# Server Side Rendering
+
+- Implementation Owner: @loks0n
+- Start Date: 31-08-2023
+- Target Date: Unknown
+
+## Summary
+
+[summary]: #summary
+
+Appwrite Auth is currently tailored towards client side rendered (CSR) applications. This RFC proposes to add support for server side rendered (SSR) applications.
+
+## Problem Statement (Step 1)
+
+[problem-statement]: #problem-statement
+
+**What problem are you trying to solve?**
+
+Many popular web frameworks, such as Next.js, Nuxt.jsm and SvelteKit, support and recommend SSR, by default. When developers attempt to use Appwrite Auth with these frameworks, they run into issues with the current implementation.
```

Comment:
```suggestion
Many popular web frameworks, such as Next.js, Nuxt.js and SvelteKit, support and recommend SSR, by default. When developers attempt to use Appwrite Auth with these frameworks, they run into issues with the current implementation.
```
Sorry 😬 

Also, I'd say the problem is not only Appwrite Auth. I've also run into issues with file uploads. I've managed to get it working, but it was [troublesome to say the least](https://github.com/TGlide/pluck-svelte/blob/main/src/lib/appwrite/storage.server.ts).